alpinelib/aws/s3.py

```diff
@@ -79,7 +79,7 @@ def move_objects(bucket, source, destination, destination_bucket='', delete_sour
 
     try:
         for obj in filter(lambda x: x.key.endswith(suffix), s3.Bucket(name=bucket).objects.filter(Prefix=source)):
-            dest_key = destination + str(obj.key)[str(obj.key).rfind('/')+1:]
+            dest_key = destination + str(obj.key).split("/", 1)[1]
```

Comment:
If there is a key with no "/", I think it will error with index out of range. Will there always be a slash? even if its at the root of a bucket?

Reply:
With the rfind, it returns "-1", so would be 0 in the end

Reply:
yes will always have slash